src/client/game.cpp

```diff
@@ -3052,7 +3052,7 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud, bool show_debug)
 	case CAMERA_MODE_THIRD_FRONT:
 		shootline.start = camera->getHeadPosition();
 		// prevent player pointing anything in front-view
-		d = 0;
+		d = d == -1 ? d : 0;
```

Comment:
Pointing should be disabled here (according to the comment), so I think this should be `d = -1;`.